src/config/config-production.ts

```diff
@@ -235,6 +239,28 @@ const cache = cacheManager.caching({
 // --------------
 
 const GITLAB_ROOT_PASSWORD = env.GITLAB_ROOT_PASSWORD ? env.GITLAB_ROOT_PASSWORD : '12345678';
+const AUTH_SERVER_BASE_URL = env.AUTH_SERVER_BASE_URL ? env.AUTH_SERVER_BASE_URL : 'https://lucify.eu.auth0.com';
```

Comment:
Not specific to this PR, but nearly all of the config values where we check for the presence of an env variable could be changed to:
```
const VARIABLE = env.VARIABLE || 'default_value';
```
instead of:
```
const VARIABLE = env.VARIABLE ? env.VARIABLE : 'default_value';
```